lib/src/base-styles/_nx-text-input.scss

```diff
@@ -122,3 +122,11 @@ textarea.nx-text-input__input {
 ::-ms-clear {
   display: none;
 }
+
+// used to duplicate the basic NxTextInput styles on a basic HTML <input> tag.
+.nx-text-input-html {
```

Comment:
What is this for

Reply:
> What is this for

Earlier testing that I should have removed: https://github.com/sonatype/sonatype-react-shared-components/pull/166/commits/3948f124e6bbf97fcaf298eccc879479251edbdd